images/slim-toolkit-debug/README.md

```diff
@@ -0,0 +1,24 @@
+<!--monopod:start-->
+# slim-toolkit-debug
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/slim-toolkit-debug` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/slim-toolkit-debug/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+*[Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+Container image with some debugging utilies included. Suitable for using as a debugging tool.
+
+## Get It!
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/debug:latest
+```
```

Comment:
This could use an update

Reply:
sorry about that I assumed that `monopod readme` already fixed that, thanks, good catch!